autowrap/PXDParser.py

```
@@ -514,8 +533,12 @@ class MethodOrAttributeDecl(object):
     def parseTree(cls, node: CVarDefNode, lines, pxd_path):
         if isinstance(node, CEnumDefNode):
             return EnumDecl.parseTree(node, lines, pxd_path)
+        try:
+            annotations = parse_line_annotations(node, lines)
+        except ValueError as e:
+            logger.warning(f"Failed to parse line annotations in {pxd_path}:{node.pos[1]}\n{e}")
+            raise ValueError(f"Failed to parse line annotations in {pxd_path}:{node.pos[1]}\n{e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)